handbook/product/pricing-features-table.yml

```diff
@@ -110,7 +110,10 @@
     - name: Audit log
       tier: Premium
       comingSoon: false
-    - name: Just-in-time provisioning
+    - name: Just-in-time (JIT) provisioning
+      tier: Premium
+      comingSoon: false
+    - name: Automated user role sync (Okta, ActiveDirectory)
```

Comment:
Wait, what is this?

Reply:
Is this referring to this one? https://github.com/fleetdm/fleet/issues/8411 

I think it would not be limited to Okta and AD... almost any SAML can do it...

Reply:
@zhumo Agreed!  Yeah this spawned out of Zach letting me know how we'd gone above and beyond just basic JIT now, and realizing we weren't listing that yet.  (My understanding: it checks every hour to put people in the right role)

Thinking on listing "Okta" and AD is just to have it be explicit for people using those things.  Could add ", or any IdP"

If this is shipped, then let's get it live.  If I'm wrong, then please stop me 😁

Reply:
I think we can say "via SAML" to not mistakenly exclude anyone. 

I'm not sure if it's every hour. I don't think we pro-actively ask the IDP. Instead, every time the IDP re-authenticates the user to Fleet, the roles get updated.

Reply:
I updated the line to be about SAML. WDYT?

Reply:
I think it has less sex appeal than when we included a few examples for folks who might not have a good understanding that Okta and AD are SAML.  Perhaps both?

Reply:
ok sounds good.